### PR TITLE
Review during July sprint

### PR DESCRIPTION
**Pull request by malvikasharan**
Lessons reviewed:

- [x] Index
- [x] introduction
- [x] Setup
- [x] RDM
- [ ] Open and version control lesson

### CODE REVIEW

File: _episodes/02-motivation.md [hunk 1]
```diff
@@ -184,27 +164,26 @@ This applies directly to the example of working on article revisions - will you
...
+The snow ball effect may be huge, and the objective of this lesson is to allow you to do **better science in less time** ( https://www.nature.com/articles/s41559-017-0160:)
 
+> As an example it was estimated that research data management  takes about 5% of your time, on the other hand, time lost due to poor data management is estimated to be 15%. 
 
 
 ### Team and community building
 
+<img src="../fig/research-foundation.jpg" alt="A house representing machine learing and AI is set upon bricks that one person is sliding below the house. On the bricks, we can read data science principles like open science, backups, reproducibiliy, and FAIR principles." width="500"/>
+
+_Data science foundations. The Turing Way project illustration by Scriberia for The Turing Way Community Shared under CC-BY 4.0 License. Zenodo. http://doi.org/10.5281/zenodo.3332807_
```
Comment (jcolomb) [inline _episodes/02-motivation.md:186]: wondering if this is the right place for this figure, (it is also in the introduction) ?

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -18,49 +18,54 @@ keypoints:
...
 In all the resources developed within this project, we consistently emphasise how computational and data science approaches can be applied while ensuring reproducibility, collaboration and transparent reporting.
-**The goal is to maintain the highest standards of research practice and integrity**.
-
 
+> **The goal is to maintain the highest standards of research practice and integrity**.
+>
 {: .testimonial}
 
-In this training material for **Managing and supervising computational Projects**, we discuss essential practices for computational reproducibility required for carrying out meaningful analyses of research datasets through data exploration, processing, visualisation and communication. 
+In this training material for **Managing Open and Reproducible Computational Projects**, we discuss essential practices for computational reproducibility required for carrying out meaningful analyses of research datasets through data exploration, processing, visualisation and communication.
```
Comment (jcolomb) [inline _episodes/01-introduction.md:65]: not sure about the title, especially the "open" part. it may scare people.

Comment (malvikasharan) [reply _episodes/01-introduction.md:65]: we can totally drop open and reproducible both
```suggestion
In this training material for learning how to manage computational projects, we discuss essential practices for computational reproducibility required for carrying out meaningful analyses of research datasets through data exploration, processing, visualisation and communication. 
```

File: _episodes/02-motivation.md (hunk 1 above)
Comment (malvikasharan) [reply _episodes/02-motivation.md:186]: Moved the images around. I can't see the preview but let me see how I an share.

Comment (jcolomb) [reply _episodes/02-motivation.md:186]: I can serve it on my computer without problems

Comment (malvikasharan) [reply _episodes/02-motivation.md:186]: Great! do feel free to move around images. There are more to add.

File: index.md [hunk 3]
```diff
@@ -4,18 +4,18 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
-<p>The <strong>Data Science for Biomedical Scientists</strong> project is funded by <a href="https://www.turing.ac.uk">The Alan Turing Institute</a> - <a href="https://www.turing.ac.uk/research/asg">AI for Science and Government</a> (ASG) Research Programme.
-The Managing Open and Reproducible Computational Projects training material covers best practices for managing and supervising computational projects in biology and related fields through data science methods, analysis, interpretation, and reporting processes.
-Through applications of scientific rigour and ethical approaches, researchers will be able to combine lessons learned in this training to design reproducible, transparent and collaborative computational projects. 
-</p>
+<p>The <strong>Managing Open and Reproducible Computational Projects<strong> training material covers best practices for managing and supervising computational projects in biology and related fields through data science methods, analysis, interpretation, and reporting processes.
```
Comment (tobyhodges) [inline index.md:7]:
```suggestion
<p>The <strong>Managing Open and Reproducible Computational Projects</strong> training material covers best practices for managing and supervising computational projects in biology and related fields through data science methods, analysis, interpretation, and reporting processes. 
```

FIxing the closing tag. Making this change should solve the problem @jcolomb observed with the index page rendering incorrectly.

Comment (malvikasharan) [reply index.md:7]: Ahh thank you Toby.

File: index.md [hunk 4]
```diff
@@ -4,18 +4,18 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
 
 {% comment %} This is a comment in Liquid {% endcomment %}
 
 > ## Prerequisites
 >
-> This resource is designed for experimental biologists and biomedical research communities, with a focus on two key professional/career groups:
+> This resource is designed for experimental biologists, biomedical researchers and adjacent communities, with a focus on two key professional/career groups:
 > * Group leaders or lab managers without prior experience with Data Science or management of computational projects
 > * Postdoc and lab scientists (next-generation senior leaders) interested in enabling the integration of computational science into biosciences
 >
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply index.md:21]: There was an issue that Toby fixed (html tag error).

File: _episodes/04-rdm.md [hunk 5]
```diff
@@ -123,14 +136,28 @@ The FAIR principles facilitate the availability of research data so that others
...
+Making data ‘FAIR’ is not the same as making it ‘open’. Accessible means that there is a procedure in place to access the data. Data should be as open as possible, and as closed as necessary. It is also important to say that the FAIR principles are aspirational: they do not strictly define how to achieve a state of FAIRness, but rather describe a continuum of features, attributes, and behaviours that will move a digital resource closer to that goal. Even though the FAIR principles have been defined to allow machines to find and use digital objects automatically, they improve the reusability of data by humans as well. The capacity of computational systems to find, access, interoperate, and reuse data, with minimal human intervention, is essential in today’s data-driven era.
 
 You can find a more detailed [overview of the FAIR principles by GO FAIR](https://www.go-fair.org/fair-principles) of what the FAIR principles recommend.
 
+### Summary of "FAIR - How To"
+
+> We have provided an additional lesson to discuss the How-Tos of FAIR principles in the context of data and software. Please see details in [](../../_extra/-4-FAIRHowTo.md).
```
Comment (malvikasharan) [inline _episodes/04-rdm.md:145]:
```suggestion
> We have provided an additional lesson to discuss the How-Tos of FAIR principles in the context of data and software. See [FAIR How-To for data and software](../../_extra/-4-FAIRHowTo.md) for detail.
```

**Pull request merged**